db/manual_compaction_test.cc

```diff
@@ -149,7 +177,121 @@ TEST_F(ManualCompactionTest, Test) {
 
   // close database
   delete db;
-  DestroyDB(dbname_, ROCKSDB_NAMESPACE::Options());
+  DestroyDB(dbname_, Options());
+}
+
+TEST_F(ManualCompactionTest, SkipLevel) {
+  DB* db;
+  Options options;
+  options.num_levels = 3;
+  options.level0_file_num_compaction_trigger = INT_MAX;
```

Comment:
Which INT_MAX is it? To be safe, you can use a large number her.

Reply:
changed to 100 and left a comment on why set to 100